src/main/java/io/cdap/plugin/http/source/common/ConfigUtil.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.http.source.common;
+
+import com.google.common.base.Strings;
+import io.cdap.cdap.etl.api.validation.InvalidConfigPropertyException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Methods for configs validation and configs conversion.
+ */
+public class ConfigUtil {
+  public static void assertIsSet(Object propertyValue, String propertyName, String reason) {
```

Comment:
are these methods going to be used anywhere except the config? If not it's better just to put them in the config class.

Reply:
done